soul-client/soul-client-http/soul-client-springmvc/src/test/java/org/dromara/soul/client/springmvc/init/SpringMvcClientBeanPostProcessorTest.java

```diff
@@ -36,28 +44,51 @@
 @RunWith(MockitoJUnitRunner.class)
 public final class SpringMvcClientBeanPostProcessorTest {
 
-    private final SpringMvcClientTestBean springMvcClientTestBean = new SpringMvcClientTestBean();
+    private static Undertow server;
+
+    private static long registerNum;
 
-    private SpringMvcClientBeanPostProcessor springMvcClientBeanPostProcessor;
+    private static SpringMvcClientBeanPostProcessor springMvcClientBeanPostProcessor;
 
-    @Before
-    public void init() {
+    private final SpringMvcClientTestBean springMvcClientTestBean = new SpringMvcClientTestBean();
+
+    @BeforeClass
+    public static void init() {
         SoulSpringMvcConfig soulSpringMvcConfig = new SoulSpringMvcConfig();
-        soulSpringMvcConfig.setAdminUrl("localhost");
+        soulSpringMvcConfig.setAdminUrl("http://127.0.0.1:58888");
         soulSpringMvcConfig.setAppName("test-mvc");
         soulSpringMvcConfig.setContextPath("test");
-        soulSpringMvcConfig.setPort(8080);
+        soulSpringMvcConfig.setPort(58889);
         springMvcClientBeanPostProcessor = new SpringMvcClientBeanPostProcessor(soulSpringMvcConfig);
+        server = Undertow.builder()
+                .addHttpListener(58888, "localhost")
+                .setHandler(path()
+                        .addPrefixPath("/soul-client/springmvc-register", httpServerExchange -> registerNum++))
+                .build();
+        server.start();
+    }
+
+    @AfterClass
+    public static void after() {
+        server.stop();
     }
 
     @Test
-    public void testSoulBeanProcess() {
+    public void testSoulBeanProcess() throws InterruptedException {
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        registerNum = 0L;
         springMvcClientBeanPostProcessor.postProcessAfterInitialization(springMvcClientTestBean, "springMvcClientTestBean");
+        countDownLatch.await(5, TimeUnit.SECONDS);
+        Assert.assertEquals(registerNum, 1L);
     }
 
     @Test
-    public void testNormalBeanProcess() {
+    public void testNormalBeanProcess() throws InterruptedException {
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        registerNum = 0L;
         springMvcClientBeanPostProcessor.postProcessAfterInitialization(new Object(), "normalBean");
+        countDownLatch.await(5, TimeUnit.SECONDS);
```

Comment:
If you don't call `countDownLatch.countDown(); ` It will always await for 5s.

Reply:
but I just call countDownLatch.await(), it is keep await and never run again .